frontend/apps/desktop/src/styles.less

```diff
@@ -1 +1,34 @@
-/* You can add global styles to this file, and also import other style files */
+@import "less/fonts-desktop.less";
+@import "less/colors-desktop.less";
+
+body {
+  margin: 0;
+  background-color: @color-bg-light;
+}
+
+p {
+  font-family: "Helvetica Neue", Helvetica, sans-serif;
+  margin: 0 !important;
+}
+
+h1 {
+  font-family: "Didot", serif;
+  color: @color-red;
+  font-size: 48px;
+  margin: 0;
+}
+
+h2 {
+  font-family: "Helvetica Neue", Helvetica, sans-serif;
+  color: @color-red;
+  font-size: 24px;
+  margin: 0;
+  font-weight: normal;
+}
+
+html, body { height: 100%; }
+body { margin: 0; font-family: "Helvetica Neue", Helvetica, sans-serif; }
```

Comment:
Wenn du die gleiche Font Family an mehrere Stellen nutzt, könntest du es auch in eine var auslagern